### ISSUE TITLE
Scribus support [feature request]

### ISSUE THREAD

**Issue by smurzin** [Sagelt/Dungeon-World#177]
It would be great if aside indd files, there would be PostScript files, so users with Linux could actually read the book.

**Comment by Anth0rx** [comment]
@Sagelt Is there any update on this issue? I'd love seeing Dungeon World in an editable free and open source format.

**Comment by lifelike** [comment]
I posted a version in org-mode format that is more editable, but lacks any of the original formatting or graphics. It might be possible to use some clever org-export settings (or pandoc) to generate nicer PDFs from that version, but never looked into it. Pandoc has some InDesign support, but not sure if it can import the styles from DW to apply to an org-mode document? https://github.com/lifelike/Dungeon-World-Org-Mode

**Comment by Anth0rx** [comment]
Well, that's good to know. Thank you very much @lifelike :+1:
Still it would be awesome to see Dungeon World documents (especially the character sheets) in an editable format.

**Comment by Anth0rx** [comment]
@Sagelt Is there a way for me to help in that situation?

**Comment by lifelike** [comment]
The rules are already in a reasonably editable (XML) format. I think the best thing you could do is add a script of some kind to turn that into what. scribus can render. I have a script to generate the org-mode version (to be released sometime in the future). It is not a very difficult format to parse. There are many formats that are more convenient to edit, but imo not worth maintaining two different plain-text formats, and maintaining some binary (scribus?) version would be even worse.

**Comment by Anth0rx** [comment]
@lifelike You are right, the text itself is editable in xml. What I mean are the character sheets, the layout, etc.

**Comment by Anth0rx** [comment]
@Sagelt It would be possible to simply save the indd files as PostScript, wouldn't it?

**Comment by nerun** [comment]
# Suggestion, use PANDOC
In Debian-like distros: `sudo aptitude install pandoc`, or any "apt" tool you like.

Usage:
1. Convert to HTML:
`pandoc filename.xml -o filename.html`
2. Convert HTML to ODT:
`pandoc filename.html -o filename.odt`
3. Or, in one single line, deleting the intermediate file at end:
`pandoc filename.xml -o filename.html && pandoc filename.html -o filename.odt && rm filename.html`

Don't convert directly to ODT, it gonna lose formatting!
